usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js

```diff
@@ -335,11 +334,6 @@
             wasLastPointOnBoard = isCurrentPointOnBoard;
             if (event.isLeftButton) {
                 drawingInDesktop = true;
-                if (!isCurrentPointOnBoard) {
```

Comment:
I don't _think_ this is the right solution - I think you simply want to return early if `!isCurrentPointOnBoard`. No?

Reply:
Ack